docs/releases/1.8.9.txt

```diff
@@ -11,3 +11,6 @@ Bugfixes
 
 * Fixed a regression that caused the "user-tools" items to display on the
   admin's logout page (:ticket:`26035`).
+
+* Fixed a crash when getting translations and when the current language has no
```

Comment:
not entirely clear what "getting translations" means. Maybe "Fixed a crash in the translations system when the current language has no translations" is sufficient?